src/PolygonClipper/StablePriorityQueue{T,TComparer}.cs

```diff
@@ -86,61 +92,66 @@ public T Peek()
     }
 
     /// <summary>
-    /// Restores the heap property by moving the item at the specified index upward.
+    /// Restores the min-heap property by moving the item at the specified index upward
+    /// through the heap until it is in the correct position. This is called after insertion.
     /// </summary>
-    /// <param name="index">The index of the item to move upward.</param>
-    private void Up(int index)
+    /// <param name="index">The index of the newly added item to sift upward.</param>
```

Comment:
Typo

Reply:
Actually, that's intentional, sift is standard terminology in [heap operations](https://en.wikipedia.org/wiki/Heap_(data_structure))